.github/workflows/upload-to-pypi.yml

```
@@ -1,70 +1,80 @@
 name: Publish release to PyPI
 
 on:
+  workflow_dispatch:
   release:
     types: [published]
 
 jobs:
-  build-wheel-windows-macos:
+  build-wheels:
     strategy:
       matrix:
-        os: [windows-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
-        python-architecture: [x86, x64]
-        exclude:
-          - os: macos-latest
-            python-architecture: x86
+        os: [windows-latest, macos-latest, ubuntu-latest]
 
     runs-on: ${{ matrix.os }}
 
     steps:
-    - name: Check out the commit
-      uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }} (${{ matrix.python-architecture }})
-      uses: actions/setup-python@v2
+
+    - name: Check out the release commit
+      uses: actions/checkout@v3
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
       with:
-        python-version: ${{ matrix.python-version }}
-        architecture: ${{ matrix.python-architecture }}
-    - name: Install prerequisites
-      run: |
-        python -m pip install --upgrade pip setuptools
-        python -m pip install wheel
-        python -m pip install twine
-    - name: Build wheel
+        platforms: arm64
+      if: runner.os == 'Linux'
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install Python packages needed for wheel build and upload
       run: |
-        python -m pip wheel --no-deps -w ./dist .
-    - name: Publish wheels to PyPI
+        python -m pip install --upgrade pip
+        python -m pip install twine wheel
```

Actually, I think `wheel` should be unnecessary here.
```suggestion
        python -m pip install twine
```